### DISCUSSION TITLE
Could someone help me test my FundMe

### DISCUSSION THREAD

**Discussion by KaneK31** [Cyfrin/foundry-full-course-cu#2686] [General]
I hope this doesn't breach some sort of rule but can someone test to see if they're able to test zksync testnet eth to this address, when I try im greeted with internal JSON-RPC error.
0x91777Fb2D365cbBFdC86f0FC5b640f2FC8f96BA2.

**Comment by EngrPips** [comment]
hmmmmm🤔
